src/serac/physics/base_physics.hpp

```diff
@@ -99,13 +93,146 @@ class BasePhysics {
    */
   virtual void completeSetup() = 0;
 
+  /**
+   * @brief Accessor for getting named finite element state fields from the physics modules
+   *
+   * @param state_name The name of the Finite Element State to retrieve
+   * @return The named Finite Element State
+   */
+  virtual const FiniteElementState& state(const std::string& state_name) = 0;
+
+  /**
+   * @brief Get a vector of the finite element state solution variable names
+   *
+   * @return The solution variable names
+   */
+  virtual std::vector<std::string> stateNames() = 0;
+
+  /**
+   * @brief Generate a finite element state object for the given parameter index
```

Comment:
There is some inconsistency between "finite element state" vs "FiniteElementState"

Reply:
Would you prefer "finite element state" in these kind of comments, or `FiniteElementState`?

Reply:
I generally prefer the class name so its easy to look up but consistency is most important. We can fix it after in a general doxygen pr.